.changeset/floppy-starts-find.md

```
@@ -0,0 +1,5 @@
+---
+"task-master-ai": minor
+---
+
+Add CLI & MCP progress tracking for parse-prd command.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)